packages/@react-stately/table/package.json

```diff
@@ -18,6 +18,7 @@
   },
   "dependencies": {
     "@babel/runtime": "^7.6.2",
+    "@react-aria/utils": "^3.13.3",
```

Comment:
This is an aria dependency in a stately package

Reply:
never again!

Reply:
ugh.... fails SSR Test